### PR TITLE
Fix for 416

### LINKED ISSUE #416: Cannot use the ConditionOperator.In Operator for Guid
Hi,

In case you are using FilterExpression and you want to filter base on records Ids, normal case you have to pass an array of string contain the Ids (string and not Guid) but while testing my code i was getting the error `The binary operator Equal is not defined for the types 'System.Guid' and 'System.String'`

### PR DESCRIPTION
**Pull request by BetimBeja**
Hello @jordimontana82 ,
this fixes #416.
Best regards,
Betim.

### CODE REVIEW

File: FakeXrmEasy.Tests.Shared/FakeContextTests/RetrieveMultiple/RetrieveMultipleTests.cs [hunk 1]
```diff
@@ -566,5 +567,26 @@ public void Should_Allow_Using_Aliases_with_Dot()
...
+        [Fact]
+        public void TheCorrectResultIsReturnedWhenUsingConditionOperatorInWithGuid()
+        {
+            var context = new XrmFakedContext();
+            context.ProxyTypesAssembly = Assembly.GetExecutingAssembly();
+            var contact = new Crm.Contact()
+            {
+                Id = Guid.NewGuid()
+            };
+            context.Initialize(contact);
```
Comment (jordimontana82) [inline FakeXrmEasy.Tests.Shared/FakeContextTests/RetrieveMultiple/RetrieveMultipleTests.cs]: Minor feedback @BetimBeja , when using Initialize() with early bound there's no need to specify the ProxyTypesAssembly manually, FakeXrmEasy will retrieve internally

Comment (jordimontana82) [reply FakeXrmEasy.Tests.Shared/FakeContextTests/RetrieveMultiple/RetrieveMultipleTests.cs]: Thanks again for this PR, been hectic after christmas but will spend more time on this again :)

Comment (BetimBeja) [reply FakeXrmEasy.Tests.Shared/FakeContextTests/RetrieveMultiple/RetrieveMultipleTests.cs]: I removed the unnecessary line of code.
I am always glad to help with this 😃

**Comment by jordimontana82** [comment]
That's approved and merged now, thx Betim!

**Pull request merged**